abuseipdb/help.md

```diff
@@ -339,6 +339,7 @@ There's a rate limit on the free API service. The following error messags `429 C
 
 # Version History
 
+* 5.0.5 - Fix none type error in `Check CIDR`
```

Comment:
@ncramer-r7 You can write out the action title normally, we only use backticks when it's referring to a specific piece of code like a variable name
```suggestion
* 5.0.5 - Fix none type error in Check CIDR action
```